### ISSUE TITLE
blog about python packages I am using

### ISSUE THREAD

**Issue by franklingu** [franklingu/franklingu.github.io#186]
agate==1.6.0
agate-dbf==0.2.0
agate-excel==0.2.1
agate-sql==0.5.2
alabaster==0.7.10
ansible==2.4.1.0
ansible-container==0.9.2
appdirs==1.4.3
asn1crypto==0.22.0
astroid==1.6.1
attrs==17.4.0
awscli==1.11.123
Babel==2.5.1
backports.csv==1.0.5
bcrypt==3.1.4
beautifulsoup4==4.5.3
better-exceptions==0.1.8
botocore==1.5.86
bs4==0.0.1
certifi==2018.1.18
cffi==1.10.0
chardet==3.0.4
cli-helpers==0.2.3
click==6.7
colorama==0.3.7
configobj==5.0.6
coverage==4.5.1
cryptography==1.9
cssselect==1.0.1
csvkit==1.0.2
Cython==0.27.3
data-utils==0.4.5
dbfread==2.0.7
decorator==4.0.11
docutils==0.13.1
EasyProcess==0.2.3
et-xmlfile==1.0.1
feedfinder2==0.0.4
feedparser==5.2.1
future==0.16.0
graphviz==0.8.2
humanize==0.5.1
idna==2.6
imagesize==0.7.1
ipdb==0.10.3
ipython==6.0.0
ipython-genutils==0.2.0
isodate==0.5.4
isort==4.3.4
jdcal==1.3
jedi==0.10.2
jieba3k==0.35.1
Jinja2==2.9.6
jmespath==0.9.3
lazy-object-proxy==1.3.1
leather==0.3.3
line-profiler==2.1.2
lxml==4.1.1
MarkupSafe==1.0
mccabe==0.6.1
memory-profiler==0.52.0
mysql-connector==2.1.6
newspaper3k==0.1.9
nltk==3.2.2
numpy==1.14.1
objgraph==3.4.0
olefile==0.44
openpyxl==2.4.8
packaging==16.8
pandas==0.22.0
paramiko==2.3.1
parsedatetime==2.4
pexpect==4.2.1
pgcli==1.8.1
pgspecial==1.9.0
pickleshare==0.7.4
Pillow==4.1.1
pkginfo==1.4.1
pluggy==0.6.0
prompt-toolkit==1.0.14
psutil==5.4.1
psycopg2==2.7.3.2
ptyprocess==0.5.1
py==1.5.2
pyarrow==0.7.0
pyasn1==0.2.3
pycparser==2.17
pyfakefs==3.4
Pygments==2.2.0
pylint==1.8.2
pymongo==3.6.0
PyNaCl==1.1.2
pyOpenSSL==17.0.0
pyparsing==2.2.0
pyrasite==2.0
pytest==3.4.2
pytest-cov==2.5.1
python-dateutil==2.7.0
python-slugify==1.2.4
pytimeparse==1.1.7
pytz==2018.3
PyVirtualDisplay==0.2.1
PyYAML==3.12
requests==2.18.4
requests-file==1.4.2
requests-toolbelt==0.8.0
retinasdk==1.0.0
rsa==3.4.2
ruamel.yaml==0.15.34
s3transfer==0.1.10
selenium==3.4.3
setproctitle==1.1.10
simplegeneric==0.8.1
simplejson==3.10.0
six==1.11.0
snowballstemmer==1.2.1
Sphinx==1.6.6
sphinxcontrib-websupport==1.0.1
SQLAlchemy==1.1.14
sqlparse==0.2.4
structlog==17.2.0
termcolor==1.1.0
terminaltables==3.1.0
tldextract==2.0.2
tldr==0.3
tqdm==4.19.4
traitlets==4.3.2
twine==1.9.1
Unidecode==0.4.21
urllib3==1.22
wcwidth==0.1.7
wrapt==1.10.11
xlrd==1.0.0
